projects/find/find.py

```diff
@@ -0,0 +1,37 @@
+import os
+import argparse
+
+"""This method is created to find all files and directories, given a specific path. Optionally, your can use -name flag
+to search only the files that match to a specific name"""
+
+
+def find(path, name=None):
+    if not os.path.exists(path):
+        return "Path doesn't exist or wrong: please try again"
+    stack = []
+    stack.append(path)
+    while len(stack) > 0:
+        stack_element = stack.pop(0)
+        if not os.path.islink(stack_element):
```

Comment:
do you really need this extra if branching here? you should check if it's a symlink, but maybe you can save some lines of code and logic if you aggregate the condition somewhere else